website/docs/reference/dbt-jinja-functions/model.md

```diff
@@ -42,6 +42,48 @@ If you're using the CLI, use [log()](/reference/dbt-jinja-functions/log) to prin
 
 </Tabs>
 
+## Batch properties for microbatch models
+
+From dbt Core v1.9, the model object includes a `batch` property (`model.batch`), which provides details about the current batch when executing an [incremental microbatch](/docs/build/incremental-microbatch) model. This property is only populated during the batch execution of a microbatch model.
+
+The following table describes the properties of the `batch` object. Note that dbt appends the property to the `model` and `batch` objects. 
+
+| Property | Description | Example |  
+| -------- | ----------- | ------- |
+| `id` | The unique identifier for the batch within the context of the microbatch model. | `model.batch.id` |
+| `event_time_start` | The start time of the batch's [`event_time`](/reference/resource-configs/event-time) filter (inclusive). | `model.batch.event_time_start` |
```

Comment:
@QMalcolm what do we mean by 'inclusive' and 'exclusive' here?

Reply:
@mirnawong1 great question ❤️ 

### Short answer
Inclusive means the value is included in the batch, exclusive means the value is excluded from the batch

### Long answer
To think specifically about inclusivity/exclusivity means with regard to a batch, consider the batch filter statement 
```
WHERE event_time >= event_time_start AND event_time < event_time_end
```
Things to note
1. The comparison of `event_time` to `event_time_start` used `>=` with the `=` part of the operator including the `event_time_start` as part of the range. This means the usage of `event_time_start` is _inclusive_ in the range selection 
2. In the comparison of `event_time` to `event_time_end` we used `<` meaning things up to, but **not equal to**, the `event_time_end` will be selected as part of the range. This means that `event_time_end` is _exclusive_ in the range selection.